### PR TITLE
declare variable from "var" to "make"

### PR DESCRIPTION
**Pull request by wangchenglong01**
## Which problem is this PR solving?
- declare variable from "var" to "make"

## Short description of the changes
- Because we have knowed the slice's length. So we can avoid using 'append' to enlarge the extra memory.

### CODE REVIEW

**Comment by wangchenglong01** [comment]
@roidelapluie 
Hello, my pr's reviewers are none. Could you please handle it or who?

**Comment by beorn7** [comment]
I would say this kind of optimization is only required in (very) hot code paths. However, this is in a test, where the gain in efficiency is irrelevant, if noticeable at all.

The original code is idiomatic and readable. With your change, it's harder to read. (The `make` construct could mean many things, so the reader has it carefully: what is created? is it length or capacity I'm setting? The previously used `append` construct, while a bit verbose, is an extremely common and well known idiom.) It is also harder to maintain. (If the length ever changes, now two code locations have to be changed. If you change only one, you have created a bug.)

In short: I think this code should stay as is.

**Comment by wangchenglong01** [comment]
> I would say this kind of optimization is only required in (very) hot code paths. However, this is in a test, where the gain in efficiency is irrelevant, if noticeable at all.
> 
> The original code is idiomatic and readable. With your change, it's harder to read. (The `make` construct could mean many things, so the reader has it carefully: what is created? is it length or capacity I'm setting? The previously used `append` construct, while a bit verbose, is an extremely common and well known idiom.) It is also harder to maintain. (If the length ever changes, now two code locations have to be changed. If you change only one, you have created a bug.)
> 
> In short: I think this code should stay as is.

I got it. Thanks for your answer.
